dags/sustainment/information_url_checker.py

```diff
@@ -0,0 +1,88 @@
+import requests
+import logging
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.python import PythonOperator
+from ckan_operators.package_operator import GetCkanPackageListOperator
+from utils_operators.slack_operators import GenericSlackOperator, task_failure_slack_alert
+
+# init hardcoded vars for these dags
+ACTIVE_ENV = Variable.get("active_env")
+CKAN_CREDS = Variable.get("ckan_credentials_secret", deserialize_json=True)
+CKAN = CKAN_CREDS[ACTIVE_ENV]["address"]
+CKAN_APIKEY = CKAN_CREDS[ACTIVE_ENV]["apikey"]
+
+with DAG(
+    'information-url-checker',
+    default_args={
+                "owner": "Yanan",
+                "depends_on_past": False,
+                "email": "yanan.zhang@toronto.ca",
+                "email_on_failure": False,
+                "email_on_retry": False,
+                "retries": 1,
+                "retry_delay": 3,
+                "on_failure_callback": task_failure_slack_alert,
+                "tags": ["sustainment"]
+            },
+    description='Check if information_url(Under dataset description section) accessible, send notification to slack on failure urls',
+    schedule_interval="@once",
```

Comment:
We should set this guy up to run weekly IMO - it will spam slack, but it will also tell us if an existing information_url suddenly gets taken down